index.js

```diff
@@ -126,7 +129,7 @@ class DatGateway extends DatLibrarian {
         // return index
         if (!address && !path) {
           res.writeHead(200)
-          res.end(welcome)
```

Comment:
I'm not sure about getting rid of the welcome message. It has some useful information other than "dat-gateway".

Maybe we could change it later to showcase more of the features like the DatArchvie API?

Reply:
hey no worries - there's some bigger changes I've got coming up, let's delay this merge while I sort a few more tings out.

Reply:
I modified the bunsen code and revert back to `res.end(welcome)` - works fine.